stockroom/cli.py

```diff
@@ -96,6 +112,35 @@ def liberate():
         click.echo("Error while attempting to release the writer lock")
 
 
+@stock.command(name="list")
+@click.option("--data", "-d", is_flag=True)
+@click.option("--model", "-m", is_flag=True)
+@click.option("--experiment", "-e", is_flag=True)
+def list_shelf(data, model, experiment):
+    stock_obj = get_stock_obj()
+    console.print_current_head(stock_obj.head)
+    if data:
+        columns = []
+        for col in stock_obj.data.keys():
```

Comment:
`.data` or `.model` or `.experiment` attributes would be `None` if the stock object is not initialized yet. For example, if somebody `cd` to a repository and run `stock list -dme` without adding stuff, it would throw error

Reply:
Yeah I did run a quick check for that and it showed no error and printed an empty table.
but now its showing (I know which is strange, I might have overlooked something)
anyways I will fix that thanks for pointing it out!